client/my-sites/promote-post-i2/components/post-item/index.tsx

```diff
@@ -86,22 +88,31 @@ export default function PostItem( { post }: { post: BlazablePost } ) {
 				</div>
 				<div className="post-item__post-data-row post-item__post-data-row-mobile">
 					<div className="post-item__stats-mobile">
-						{ sprintf(
```

Comment:
I think it will be easier to read if we change how are we checking this conditions.
We have these fields:
Price, SKU, visitors, comments, likes.

I think it will be easier to read if we have id conditions, instead of if/else in here. Let me show you what I mean.
Currently, we only need to show the Price and SKu for Woo, so it will be easier to read if we only have this in the dom: 
```
{isRunningInWooStore && (
   /// Show Price and SKU, and append the separator dot
)}
```
Then, for all cases we will be showing visitors, so you don't need a condition there.

Lastly, we can put a condition for not WOo to hide the other two fields.

If you write it this way, another dev doesn't have to try to guess what props are being rendered, checking the else condition on all of those props. And we save duplicating code by not having visitors in two places.